swirl_proc_macro/src/background_job.rs

```diff
@@ -79,39 +79,39 @@ impl BackgroundJob {
         if let Some(constness) = constness {
```

Comment:
The only thing I have to comment is that I'd prefer this commit to be squashed onto the first. Mainly to allow `git bisect` use and also to provide a full view of the change, as a logical unit.